src/services/users.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const { v4: uuidv4 } = require('uuid');
+
+let users = [];
+// let count = 0;
+
+function getAll() {
+  return users;
+}
+
+function getById(userId) {
+  const foundUser = users.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+function create(name) {
+  const newUser = {
+    id: uuidv4(),
+    name,
+  };
+
+  users.push(newUser);
+
+  // count++;
+
+  return newUser;
+}
+
+function remove(userId) {
+  users = users.filter(user => user.id !== userId);
+}
+
+function update({ id, name }) {
```

Comment:
Missing validation for the 'id' and 'name' parameters. It should check if the provided id and name are not empty or undefined before updating the user.

Reply:
Function updateUser calls getById which return null if id is invalid or user wasn't found. Name check was added now